lib/dependencies/inspect-implementation.ts

```diff
@@ -12,8 +12,12 @@ export function getMetaData(
   root: string,
   targetFile: string
 ) {
+  // if we're dealing with a Pipfile, lets not assume its in CWD, this
```

Comment:
Reduce comments here. `getPythonEnv` can have better description.
```suggestion
  // Specify correct env for pipfile 
```

Reply:
Going to remove all comments as they were written by original dev